domain/objectstore/state/types.go

```diff
@@ -9,11 +9,10 @@ import coreobjectstore "github.com/juju/juju/core/objectstore"
 type dbMetadata struct {
 	// UUID is the uuid for the metadata.
 	UUID string `db:"uuid"`
-	// Hash is the hash of the object.
-	Hash string `db:"hash"`
-	// HashTypeID is the id of the type of hash used to generate the hash. It
-	// can be looked up in object_store_metadata_hash_type.
-	HashTypeID uint `db:"hash_type_id"`
```

Comment:
Since we dropped this, I stand behind my previous comment about naming the fields to include the type and size of the hash.

Reply:
Note: we're only dropping it for objectstore. We don't have a nice way to represent a way to ensure that we always have 2 hashes per row in SQL, not without inverting the relationships :thinking:

Reply:
Sometimes another column is fine. I think this is one of those cases.

Reply:
The column name will be the hash type.